db/db_tailing_iter_test.cc

```diff
@@ -50,94 +55,125 @@ TEST_P(DBTestTailingIterator, TailingIteratorSingle) {
 }
 
 TEST_P(DBTestTailingIterator, TailingIteratorKeepAdding) {
-  CreateAndReopenWithCF({"pikachu"}, CurrentOptions());
+  if (mem_env_ || encrypted_env_) {
+    ROCKSDB_GTEST_SKIP("Test requires non-mem or non-encrypted environment");
```

Comment:
Not 100% sure, but I think ROCKSDB_GTEST_BYPASS is preferable to ROCKSDB_GTEST_SKIP

Reply:
Our tests contains both ROCKSDB_GTEST_BYPASS and ROCKSDB_GTEST_SKIP. I'll change it to ROCKSDB_GTEST_BYPASS (added in this PR).